MC/bin/o2dpg_sim_workflow.py

```diff
@@ -529,7 +529,8 @@ def getDPL_global_options(bigshm=False):
       else: #pPb?
          INTRATE=200000 #Hz ???
 
-   simsoption=' --sims ' + ('bkg,'+signalprefix if doembedding else signalprefix)
+   # TOF -> "--use-ccdb-tof" (alternatively with CCCDBManager "--ccdb-tof-sa")
+   simsoption=' --sims ' + ('bkg,'+signalprefix if doembedding else signalprefix) + ' --use-ccdb-tof '
```

Comment:
I am not a big fan of putting a TOF-specific command-line options to all digitizers. Would you be able to customize just the TOF digitizer line?

Reply:
Ok, let me try

Reply:
done